### ISSUE TITLE
Suggestion: Date Organization for Events

### ISSUE THREAD

**Issue by koronkowy** [phildini/stayinghomeclub#41]
The list of events is inevitably going to get larger so I suggest that the events get organized by date of event, at least,  by month for added visability. And events without clear dates can make it into a general 2020 category.

Something like:

2020 ----
March ---
<event canceled>
<event canceled>

April ---
<event canceled>
<event canceled>

**Comment by vielmetti** [comment]
Agreed with this - it's going to take some refactoring - the first pass might just be adding dates to the ones where we know dates, then moving them around.

**Comment by koronkowy** [comment]
> Agreed with this - it's going to take some refactoring - the first pass might just be adding dates to the

The organization I went with was by the original date of event. Almost all of the events I was able to find this via the sources attached to the event. Any that I could not ascertain, I put them in the bulk 2020 section. :>

**Comment by trafnar** [comment]
This could be done programatically with my PR #65

**Comment by phildini** [comment]
Hi! I have for sure come around on this as the list has gotten longer. I'm definitely open to someone doing this work, if they're interested.
